packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -139,7 +149,31 @@ export class RadioGroup extends FASTElement {
             this.parentToolbar !== undefined && this.parentToolbar !== null;
     }
 
+    public disconnectedCallback(): void {
+        this.removeEventListener("keydown", this.keydownHandler);
```

Comment:
If these event handlers were added directly to the template itself, you wouldn't need to have manual code to remove them.

Reply:
moved handlers to template, needed to return true on focusOutHandler but all working again now